app/models/game_appointment.rb

```diff
@@ -0,0 +1,47 @@
+class GameAppointment < ApplicationRecord
+  include YearlyModel
+
+  alias_attribute :white_player, :attendee_one
+  alias_attribute :black_player, :attendee_two
+  belongs_to :attendee_one, class_name: "Attendee",  foreign_key: "attendee_one_id"
+  belongs_to :attendee_two, class_name: "Attendee",  foreign_key: "attendee_two_id"
+  belongs_to :round
+
+  validates :attendee_one, presence: true, uniqueness: { scope: :round,
+    message: "players can only play in one game per round" }
```

Comment:
![image](https://user-images.githubusercontent.com/176993/42851019-78346e28-89f8-11e8-8399-dcb6239b9bfe.png)

Pretty sure I'm the one responsible for this issue -- it'd be great if we could get the name of the player that triggers this error. This message on its own is very hard to debug.